copy_test.go

```diff
@@ -658,6 +658,178 @@ func TestCopy_WithOptions(t *testing.T) {
 	}
 }
 
+func TestCopy_WithPlatformFilterOptions(t *testing.T) {
```

Comment:
Do we have a test case where `ErrUnsupported` is returned?

Reply:
Added👌